app/src/main/java/woowacourse/payments/ui/add/AddPaymentCardActivity.kt

```diff
@@ -12,21 +12,36 @@ class AddPaymentCardActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         enableEdgeToEdge()
+
+        val editingCardId = intent.getStringExtra(EXTRA_CARD_ID)
+        val isEditingFromIntent = editingCardId != null
+
         setContent {
             AndroidpaymentsTheme {
                 AddPaymentCardScreen(
                     onBack = { finish() },
-                    onSave = { paymentCard ->
-                        setResult(RESULT_OK, Intent().putExtra(EXTRA_CARD, paymentCard))
+                    onSave = {
+                        val resultIntent =
+                            Intent().apply {
```

Comment:
```suggestion
                           intent.apply {
```
nit: Intent객체를 안써도 될 것 같네요😅

Reply:
아핳 그렇네요 😅
수정했습니다!